src/components/navigationLoader/NavigationHeader.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+
+export class NavigationHeader extends React.Component<any, any> {
```

Comment:
I would name this something like `NavigationLoadingHeader` or `NavigationHeaderLoading`, since there should be a real `NavigationHeader` component later on.